content/en/blog/_posts/2022-05-13-grpc-probes-in-beta.md

```diff
@@ -125,7 +125,7 @@ spec:
         port: 5000
 ```
 
-If the file called `test.yaml`, you can create the pod and check it's status.
+If the file called `test.yaml`, you could create the pod and check its status.
```

Comment:
This sentence doesn't still make sense. It's either "In the file called test.yaml" OR we need to rephrase it to something else for the entire sentence to convey meaning.

Reply:
Yes, it is changed to:
```
In the file called `test.yaml`, you can create the pod and check its status.
```